cultcargo/images/python-astro/Dockerfile

```diff
@@ -1,29 +1,28 @@
-FROM {REGISTRY}/base-cult:base-{BUNDLE_VERSION}
+FROM {REGISTRY}/python-base:{VERSION}-{BUNDLE_VERSION}
 
-ENV DEBIAN_FRONTEND=noninteractive
-
-RUN apt-add-repository -y ppa:deadsnakes/ppa
-
-RUN apt-get update && apt-get install apt-utils && \
-    apt-get install -y casacore-data casacore-tools python{VERSION}-dev python{VERSION}-distutils \
-    python3-pip python3-pytest python3-matplotlib curl\
-    && rm -rf /var/lib/apt/lists/* \
-    && rm -rf /var/cache/apt/archives 
+# add useful astro stuff
+RUN python{VERSION} -m pip install --no-cache-dir \
+    python-casacore astropy astroplan regions astro-tigger-lsm \
+    owlcat dask-ms scipy omegaconf bdsf msutils \
+    stimela katbeam
 
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python{VERSION} 1
+# set XDG cache (normally ~/.cache)
```

Comment:
Perhaps add a more comprehensive comment explaining the origin of this addition.